### PR TITLE
Added getPrice function

### PR DESCRIPTION
**Pull request by danielvandenberg95**
In draft as enums are not complete yet. Basic features are in though.

### CODE REVIEW

**Comment by danielvandenberg95** [comment]
https://github.com/icastillejogomez/degiro-api/issues/19

**Comment by Chavithra** [comment]
Hi there,

I see a hard coded "userToken" in the code.

You might want to remove that from the project history.

**Comment by danielvandenberg95** [comment]
> Hi there,
> 
> I see a hard coded "userToken" in the code.
> 
> You might want to remove that from the project history.

Thanks, done. I assumed/thought it was the same across every account as it still worked after I logged out. Should be removed.

**Comment by bdoubleu86** [comment]
I am getting a Cross-Origin error.

Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt

Any idea?

**Comment by danielvandenberg95** [comment]
> I am getting a Cross-Origin error.
> 
> Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt
> 
> Any idea?

Hi bdoubleu86, please open a new issue next time instead of hijacking a completely different and unrelated issue. As for your problem, see https://github.com/icastillejogomez/degiro-api/issues/23.

**Comment by SPecoul** [comment]
Hi there,

I'm looking forward to this feature, and I was wondering what is preventing from accepting this P.R ? 
Does it have some remaining work to do ?
How can we help to speed up the merge ? 
(sorry, I'm quite new to open source project)

**Comment by icastillejogomez** [comment]
@danielvandenberg95 did you review this code? How do you think about this PR?

File: src/types/GetPriceOptionsType.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export enum GetPriceResolution {
+  PT1D = 'PT1M',
```
Comment (Chavithra) [inline src/types/GetPriceOptionsType.ts:2]: Here are some examples of `resolution` in `Protocol Buffers` format : 

```protobuf
  enum Resolution {
    PT1S = 0;
    PT15S = 1;
    PT30S = 2;
    PT1M = 3;
    PT5M = 4;
    PT15M = 5;
    PT30M = 6;
    PT60M = 7;
    PT1H = 8;
  }
```

File: src/types/GetPriceOptionsType.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+export enum GetPricePeriod {
+  P1D = 'P1D',
```
Comment (Chavithra) [inline src/types/GetPriceOptionsType.ts:6]: Here are some examples of `period` in `Protocol Buffers` format : 

```protobuf
  enum Period {
    P1D = 0;
    P1W = 1;
    P1M = 2;
    P3M = 3;
    P6M = 4;
    P1Y = 5;
    P3Y = 6;
    P5Y = 7;
    P10Y = 8;
    YTD = 9;
  }
```

**Review by Chavithra — APPROVED** [review]
Hello there,

I have suggested some : 
- resolution
- period

The ideal would be to let the users override these two parameters if they want to.

Otherwise this looks fine to me.

**Comment by martijndierckx** [comment]
Hi all

This PR has been opened for quite some time.
What needs to be done for it to be merged?

**Pull request closed without merging**